src/App.css

```diff
@@ -12,18 +12,21 @@ body {
   width: 500px;
   border-radius: 16px;
   display: flex;
-  flex-directon: column;
+  flex-direction: column;
```

Comment:
Good catch. However, I'm still seeing vertical space that is too much, causing the page to have a scrollbar. The reason why is because you're merging 2 body CSS sets of styling together. Please check index.css line 31 (`min-height: 100vh;`). This is being applied first and then line 6 of App.css (`padding: 15%;`) is also being applied. If you remove `padding: 15%` then the scrollbar issue is fixed. `vh` stands for view height in terms of a percentage. So `100vh` means 100% of the view's height.
![image](https://github.com/bpb2008/GuessANumber/assets/152167/a01eb1ab-de7a-4d9c-8539-9e53921c5f05)
Remember to use right-click -> Inspect and then select your HTML element to check out its styling in Chrome to figure out issues like this. I noticed that both a height style and a padding style were applied to body. Then I checked the code and sure enough, both index.css and App.css were applying styles to body. 

To reduce confusion, try to apply styles to each element in one place. It's harder to see the styling in the code if styling is being applied in different files.